<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAbout</class>
 <widget class="QDialog" name="DlgAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rms_analyze.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>NetShark</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label3">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="rms_analyze.qrc">:/images/shark.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label1">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>11</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Author:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label2">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>11</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Version:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="authorLabel">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>11</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>George.Song</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="versionLabel">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>11</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>1.0.0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Plugins</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Version</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Author</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Description</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>OK</string>
         </property>
         <property name="icon">
          <iconset resource="rms_analyze.qrc">
           <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rms_analyze.qrc"/>
 </resources>
 <connections/>
</ui>
